src/components/item/settings/ThumbnailSetting.tsx

```diff
@@ -143,9 +142,8 @@ const ThumbnailSetting = ({ item }: Props): JSX.Element | null => {
           // TODO: fix type
           url={
             imgUrl ??
-            (item as EmbeddedLinkItemTypeRecord)?.extra?.[
-              ItemType.LINK
-            ]?.thumbnails?.first()
+            (item as EmbeddedLinkItemType)?.extra?.[ItemType.LINK]
```

Comment:
First check the type and then the type should be inferred correctly.
```suggestion
            (item.type === ItemType.LINK ? item.extra.[ItemType.LINK]
```